packages/sdk/src/utils/chain-constants.ts

```diff
@@ -83,8 +83,9 @@ export const DEPOSIT_CONFIRMATION_BLOCKS: {
   [L2ChainID.OPTIMISM_HARDHAT_LOCAL]: 2 as const,
   [L2ChainID.OPTIMISM_HARDHAT_DEVNET]: 2 as const,
   [L2ChainID.OPTIMISM_BEDROCK_ALPHA_TESTNET]: 12 as const,
-  [L2ChainID.BASE_GOERLI]: 12 as const,
-  [L2ChainID.BASE_MAINNET]: 50 as const,
```

Comment:
these are incorrect, we switched goerli to a 25 deposit conf time to avoid full node halts when goerli has deep reorgs